Signal/src/util/OWSScreenLockUI.m

```diff
@@ -64,6 +65,14 @@ - (void)observeNotifications
                                              selector:@selector(applicationWillResignActive:)
                                                  name:OWSApplicationWillResignActiveNotification
                                                object:nil];
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(applicationWillEnterForeground:)
+                                                 name:OWSApplicationWillEnterForegroundNotification
```

Comment:
`Screen Lock` should reflect "entered background".  `Screen Protection` should reflect "became inactive."